recipes/thrift-cpp/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: thrift-cpp
+  version: "0.9.3"
+
+source:
+  fn: thrift-0.9.3.tar.gz
+  url: http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
+  md5: 88d667a8ae870d5adeca8cb7d6795442
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - boost
```

Comment:
Though sometimes Mac needs some convincing that C++11 is ok. 😉  Here is [something]( https://github.com/conda-forge/libsigcpp-feedstock/blob/master/recipe/build.sh#L9-L17 ) that may be needed to get `clang` to work.

If you need gomp and/or fortran too, then we need the `gcc` package.